python/dolfinx/plotting.py

```diff
@@ -6,7 +6,7 @@
 
 import os
 import warnings
-
+import numba
```

Comment:
Numba shouldn't become a DOLFIN-X dependency.

Reply:
Removing this